doozer/tests/cli/test_gen_payload.py

```diff
@@ -209,15 +211,17 @@ def test_detect_inconsistent_images(self):
         self.assertEqual(gpcli.assembly_issues, ["stuff"])
 
     @patch("doozerlib.cli.release_gen_payload.PayloadGenerator.find_payload_entries")
-    def test_generate_payload_entries(self, pg_fpe_mock):
+    @patch("doozerlib.cli.release_gen_payload.PayloadGenerator.embargo_issues_for_payload")
```

Comment:
previously there was nothing in this method to inspect the output of `find_payload_entries`, which was just mocked with dummy responses; you added a call to `embargo_issues_for_payload` which actually look at those, so either need to come up with better responses, or mock that too.